test/unittests/eof_validation_test.cpp

```diff
@@ -230,6 +230,6 @@ TEST(eof_validation, EOF1_terminating_instructions)
                                    opcode == OP_INVALID || opcode == OP_SELFDESTRUCT) ?
                                    EOFValidationError::success :
                                    EOFValidationError::missing_terminating_instruction);
-        EXPECT_EQ(validate_eof(container), expected) << hex(code);
+        EXPECT_EQ(validate_eof(container, EVMC_CANCUN), expected) << hex(code);
```

Comment:
We still want to test `validate_eof` for Shanghai, better skip opcodes that are defined since Cancun on line 222.

It could be also a loop over revisions starting from Shanghai, but we want to drop this terminating instruction requirement anyway, so I'd say just "TODO iterate for all EOF revisions" is enough too.